cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactRepository.java

```diff
@@ -592,7 +592,6 @@ private ArtifactClasses inspectArtifact(Id.Artifact artifactId, File artifactFil
     } else {
       return ArtifactClasses.builder()
         .addApps(artifactClasses.getApps())
-        .addDatasets(artifactClasses.getDatasets())
```

Comment:
Does this PR reverts a lot of the changes made in the previous PRs for this feature? (6012 and 6038)? or maybe it just seems so to me...